src/tests/ftest/launch.py

```diff
@@ -2202,6 +2194,15 @@ def process(self, test, repeat, stop_daos, archive, rename, jenkinslog, core_fil
                 "depth": 1,
                 "timeout": 900,
             }
+
+            # Get the core file pattern information
+            try:
+                core_files = self._get_core_file_pattern(
+                    test.host_info.servers.hosts, test.host_info.servers.hosts, process_cores)
+            except LaunchException:
+                message = "Error obtaining the core file pattern information"
+                return self.get_exit_status(1, message, "Setup", sys.exc_info())
```

Comment:
Since we're in the process of running each test file and processing the results, we don't want to exit launch if we we encounter this error.  Instead we should mark the execution of the test file as failed:
```suggestion
                self._fail_test(self.result.tests[-1], "Process", message, sys.exc_info())
                return_code |= 256
                core_files = {}
```

Reply:
Done.